sdcm/utils/version_utils.py

```diff
@@ -414,6 +414,33 @@ def resolve_latest_repo_symlink(url: str) -> str:
     return resolved_url
 
 
+def get_specific_tag_of_docker_image(docker_repo: str):
+
+    def get_disget(tag_dict):
+        for image in tag_dict['images']:
+            if image['architecture'] == arch:
+                return image['digest']
+        return None
+
+    arch = 'amd64'
+    url = 'https://hub.docker.com/v2/repositories/{}/tags/{}'
+    docker_latest = requests.get(url.format(docker_repo, 'latest')).json()
+
+    latest_digest = get_disget(docker_latest)
+    for page_number in range(1, 5):
```

Comment:
In case of "scylla" dockerhub repo 5 pages may be not enough:
![2022_06_28--scylla-latest-docker-image-age](https://user-images.githubusercontent.com/3796058/176177345-fca80f15-a180-437f-ab17-611aff697d5d.png)

So, if we look for the specific tag then we must assume walking through all the pages if needed.